src/Formatter/Example.php

```diff
@@ -37,7 +37,6 @@ public function format(OutlineNode $scenario)
             return;
         }
 
-        // TODO: refactor this part
```

Comment:
The `TODO` is in place because there is a need for it to be fixed, not for it to be removed